src/connector/jdbc/src/main/java/com/taosdata/jdbc/rs/RestfulStatement.java

```diff
@@ -95,6 +95,10 @@ public boolean execute(String sql) throws SQLException {
 
     private String getUrl() throws SQLException {
         String dbname = conn.getClientInfo(TSDBDriver.PROPERTY_KEY_DBNAME);
+        String protocol = "http";
+//        if (conn.isUseSsl()) {
+//            protocol = "https";
+//        }
```

Comment:
why not use https ?

Reply:
Because TDengine not support ssl, this may mislead users.